src/zorak/utilities/cog_helpers/_embeds.py

```diff
@@ -407,33 +407,6 @@ def embed_spammer_warn(channel1, channel2):
     return embed
 
 
-def embed_suggestions(author, question):
```

Comment:
Was embed_suggestions and embed_suggestions_error also removed in this commit?

Reply:
Shouldn't have been to my knowledge, this pr was created only to do a little bit of formatting cleanup with embed_leave and embed_verified_success.